.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,20 @@
+name: Run Tests using Docker
+
+on:
+  push:
+    branches: [ github-action ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
does this matter wrt the `centos:7` below?

Reply:
No - the ubuntu machine just runs the Docker image so we end up running centos with a ubuntu kernel. :)